app/lib/features/chat/widgets/member_list_entry.dart

```diff
@@ -0,0 +1,195 @@
+import 'package:acter/common/dialogs/pop_up_dialog.dart';
```

Comment:
this is mainly a copy of the same for spaces, right? Isn't there any way we can reuse the existing code rather than having a second copy where just space -> convo?

Reply:
There is if we provide optional parameter of ```Convo``` in already written widget. And apply additional checks to determine if its meant for ```Space``` or ```Conversation```.

Reply:
yeah, I think I prefer that one. Just have it accept both `space` and `convo` optionally and check that at exactly one is checked and then just everywhere do `space != null ? space.function() : convo.function()`. that should be fine enough :) . I can see that we might later have to extend that for use-case-specific things, but for now that is good enough.